test/benchmarks/perftest.pl

```diff
@@ -85,8 +86,8 @@
 
 if(!$is_baseline)
 {
-    open(IN,"$basefile") or die "Couldn't open $basefile for reading.";
-    while(<IN>)
+    open my $IN, '<', $basefile or die "Couldn't open $basefile for reading.";
```

Comment:
Nope you can keep the parens, the only reason I don't is that it makes it a little clearer the my is applying in the surrounding scope on the open. Not necessary for strict at all.